<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testcontainer</class>
 <widget class="QDialog" name="testcontainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testcontainer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_layer_move">
       <property name="text">
        <string>order a layer</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="13" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_request">
       <property name="text">
        <string>get info</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_QTV_5">
       <property name="text">
        <string>Grid measure</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_geo_selected_marks">
       <property name="text">
        <string>selected_marks</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_layers">
       <property name="text">
        <string>enum layers</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_autodl">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_geo_displayMod">
       <property name="text">
        <string>display mod</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_adds">
       <property name="text">
        <string>address</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_line">
       <property name="text">
        <string>add a line</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QPushButton" name="pushButton_QTV_default_style">
       <property name="text">
        <string>default_style</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_resource">
       <property name="text">
        <string>res save/load</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_polygon">
       <property name="text">
        <string>add a polygon</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_grid_enable">
       <property name="text">
        <string>measure on/off</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_QTV_6">
       <property name="text">
        <string>Plugin test:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_geo_selectionMod">
       <property name="text">
        <string>selection mod</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_QTV_2">
       <property name="text">
        <string>Navigate</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_geo_del_sel">
       <property name="text">
        <string>delete selection</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_xml">
       <property name="text">
        <string>marks save/load</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_navigate">
       <property name="text">
        <string>navigate</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_QTV_7">
       <property name="text">
        <string>geo marker</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_cache">
       <property name="text">
        <string>cache Folder</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_QTV_3">
       <property name="text">
        <string>Layer control</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_grid_getPolygon">
       <property name="text">
        <string>get polygon</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_QTV_4">
       <property name="text">
        <string>Plugin test:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_qtv">
       <property name="text">
        <string>background osm conn</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="pushButton_QTV_test_geo_clear_sel">
       <property name="text">
        <string>clear selection</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_mark">
       <property name="text">
        <string>add marks</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QPushButton" name="pushButton_QTV_test_10000">
       <property name="text">
        <string>10000 marks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>AxMap1</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QAxWidget" name="axWidget_map1"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>AxMap2</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QAxWidget" name="axWidget_map2">
           <property name="sizePolicy" stdset="0">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="WidgetMap">
        <attribute name="title">
         <string>WidgetMap</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="qtwidget_planetosm" name="osmmap" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView_QTV_msg">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>128</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_QTV_mouseMove">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>qtwidget_planetosm</class>
   <extends>QWidget</extends>
   <header>qtwidget_planetosm.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
